UPDATING.md

```diff
@@ -59,6 +59,10 @@ https://developers.google.com/style/inclusive-documentation
 
 -->
 
+### New permission can_dag_autocomplete
+The new dag autocomplete end point requires permissions to be synchronised and updated for roles other than Admin. Please run `airflow sync_perm`,
```

Comment:
sync_perm is run automatically at webserver start up, so this part isn't needed.

Reply:
Happy to take that bit out. I have that setting turned off so thought it could possibly trip people up